pkg/virt-controller/services/template.go

```diff
@@ -503,6 +510,21 @@ func getPortsFromVMI(vmi *v1.VirtualMachineInstance) []k8sv1.ContainerPort {
 	return ports
 }
 
+func getMultusinterfaceList(vmi *v1.VirtualMachineInstance) string {
+	ifaceList := ""
+
+	for _, network := range vmi.Spec.Networks {
+		if network.Multus != nil {
+			ifaceList += network.Name + ","
```

Comment:
that would create a list in format "net1,net2,", is Multus OK with leading comma? You can keep ifaceList as []string and then join it using strings.Join(ifaceList, ",")